about.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet"> 
+    <script src="https://kit.fontawesome.com/1673b54288.js" crossorigin="anonymous"></script>
+    <title>Capstone Project - About Page</title>
+  </head>
+  <body>
+    <header id="aboutheader">
+        <div class="curtainmenu">
+            <div class="closediv">
+                <a id="closebutton" href="#"><i class="fa-solid fa-x fa-xl closer"></i></i></a>
+            </div>
+            <nav class="menuitems">
+                <a class="menuitem" href="https://asdt560.github.io/Capstone-HTML-CSS-Basic-JS/index.html">Home</a>
+                <a class="menuitem" href="#">Program</a>
+                <a class="menuitem" href="#">Join</a>
+                <a class="menuitem" href="#">Sponsor</a>
+                <a class="menuitem" href="#">News</a>
+            </nav>
+        </div>
+        <section id="topheader">
+            <div id="topheaderitems">
+                <a href="https://www.facebook.com/">
+                    <i class="fa-brands fa-facebook-f"></i>
+                </a>
+                <a href="https://twitter.com">
+                    <i class="fa-brands fa-twitter"></i>
+                </a>
+                <p>Language</p>
+                <p>My Page</p>
+                <p>Logout</p>
+            </div>
+        </section>
+        <section class="bottomheader">
+            <div id="mobilemenu">
+                <a id="mobilemenubutton" href="#">
+                    <img src="./images/mobile_menu.png">
+                </a>
+            </div>
+            <a id="logolink" href="https://asdt560.github.io/Capstone-HTML-CSS-Basic-JS/index.html">
+                <img id="logoimglink" src="./images/CCC_logo-removebg-preview.png">
+            </a>
+            <nav id="desktopmenu">
+                <a class="menuitem" href="https://asdt560.github.io/Capstone-HTML-CSS-Basic-JS/index.html">Home</a>
+                <a class="menuitem" href="#">Program</a>
+                <a class="menuitem" href="#">Join</a>
+                <a class="menuitem" href="#">Sponsor</a>
+                <a class="menuitem" href="#">News</a>
+            </nav>
```

Comment:
### You did a great job, your home page looks like the design provide but in the desktop version there is no way to come back to your home page please add the link to your home page on your about page 
below I provide clear information please make sure you implement it as required [ here ](https://github.com/elad237/capstone-project/pull/1/files#r1007155816) by the previous reviewer:grinning:  :grinning:  :grinning: 
- [ ] Kindly make sure your project contains a `Home` page and `About` page.
- [ ] It's possible to navigate between the pages, there's a link from the `Home` page to the `About` page and the opposite too. it means on your home page you will have the link to your about page and on the about page, you will have the link to your home page 
- [ ] Both pages contain the same navbar and footer.

Check the  [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) to see the style and the container of your about page :+1: :grinning: :grinning: :grinning: